src/raven/log.cr

```diff
@@ -0,0 +1,5 @@
+require "log"
+
+module Raven
+  Log = ::Log.for(self)
```

Comment:
Codacy found an issue: [Constant name should be screaming-cased: LOG, not Log](https://app.codacy.com/manual/Sija/raven.cr/pullRequest?prid=5727087)